processor/configuration/src/main/java/com/redhat/service/smartevents/processor/resolvers/custom/SlackActionResolver.java

```diff
@@ -23,8 +24,17 @@ public Action resolve(Action action, String customerId, String bridgeId, String
         resolvedAction.setParameters(action.getParameters().deepCopy());
         resolvedAction.setType(KafkaTopicAction.TYPE);
 
+        ObjectNode actionParameters = resolvedAction.getParameters();
+
         String connectorTopicName = gatewayConfiguratorService.getConnectorTopicName(processorId);
-        resolvedAction.getParameters().set(KafkaTopicAction.TOPIC_PARAM, new TextNode(connectorTopicName));
+        actionParameters.set(KafkaTopicAction.TOPIC_PARAM, new TextNode(connectorTopicName));
+
+        actionParameters.set(KafkaTopicAction.BROKER_URL, new TextNode(gatewayConfiguratorService.getBootstrapServers()));
+        actionParameters.set(KafkaTopicAction.CLIENT_ID, new TextNode(gatewayConfiguratorService.getClientId()));
+        actionParameters.set(KafkaTopicAction.CLIENT_SECRET, new TextNode(gatewayConfiguratorService.getClientSecret()));
+        actionParameters.set(KafkaTopicAction.SECURITY_PROTOCOL, new TextNode(gatewayConfiguratorService.getSecurityProtocol()));
```

Comment:
Is it possible to move this to a common abstract or similar? It is not specific to `SlackActionResolver`.
Moreover I don't understand why for example this change applies to `Slack` but not to [`AwsLambdaActionResolver`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/0e9f7f859d4266fc41990419d1a92353f84d2370/processor/configuration/src/main/java/com/redhat/service/smartevents/processor/resolvers/custom/AwsLambdaActionResolver.java#L28) etc